### PR TITLE
docs: decomposing editor page

### LINKED ISSUE #3220: 📚 - Decompose editor page
### What is the affected URL?

_No response_

### Description

image

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

Closes #3220

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/app/app.routes.ts [hunk 1]
```diff
@@ -951,14 +951,125 @@ export const ROUTES = [
...
+    // EDITOR
     {
         path: `components/editor`,
         loadChildren: async () =>
-            (await import(`../components/editor/editor.module`)).ExampleTuiEditorModule,
+            (await import(`../components/editor/getting-started/editor.module`))
+                .ExampleTuiEditorModule,
         data: {
-            title: `Editor`,
+            title: `Editor[Getting started]`,
```
Comment (waterplea) [inline projects/demo/src/modules/app/app.routes.ts:961]: Let's change format to something more human, like `Editor — Getting started`. Same for the rest of Editor pages.

File: projects/demo/src/modules/components/editor/getting-started/editor.template.html [hunk 2]
```diff
@@ -4,153 +4,38 @@
     type="components"
 >
     <ng-template pageTab>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/getting-started/editor.template.html:6]:
```suggestion
    <ng-template pageTab="Setup">
```

File: projects/demo/src/modules/app/pages.ts [hunk 3]
```diff
@@ -374,6 +374,78 @@ export const pages: TuiDocPages = [
...
         ],
     },
+    {
+        section: $localize`Components`,
+        title: `Editor`,
+        subPages: [
+            {
+                section: $localize`Components`,
+                title: `Getting started`,
+                keywords: `wysiwyg, редактор, текст, html, rich, text`,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/pages.ts:384]: Screenshot 2022-12-19 at 10 15 22

File: projects/demo/src/modules/components/editor/nested-groups/editor-nested-groups.component.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {TuiDocExample} from '@taiga-ui/addon-doc';
+import {defaultEditorExtensions, TUI_EDITOR_EXTENSIONS} from '@taiga-ui/addon-editor';
+
+@Component({
+    selector: `editor-nested-groups`,
+    templateUrl: `./editor-nested-groups.component.html`,
+    encapsulation: ViewEncapsulation.None,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/nested-groups/editor-nested-groups.component.ts:9]: why do all new components have `ViewEncapsulation.None` ?

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
```
1) Examples images
       Preview
         preview display of images:
     AssertionError: Timed out retrying after 4000ms:
     Expected to find element: `#preview-image`, but never found it.
      at Context.eval (webpack:///./cypress/tests/desktop/addon-editor/editor-img.cy.ts:24:15)
```

File: projects/demo/src/modules/app/app.routes.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/app.routes.ts]: fixed

File: projects/demo/src/modules/components/editor/getting-started/editor.template.html (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/components/editor/getting-started/editor.template.html]: fixed

File: projects/demo/src/modules/app/pages.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/pages.ts]: fixed

File: projects/demo/src/modules/components/editor/nested-groups/editor-nested-groups.component.ts (hunk 4 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/components/editor/nested-groups/editor-nested-groups.component.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**